docs/best-practices/api-design.md

```diff
@@ -408,6 +408,9 @@ It is highly unlikely that a web API will remain static. As business requirement
 
 Versioning enables a web API to indicate the features and resources that it exposes, and a client application can submit requests that are directed to a specific version of a feature or resource. The following sections describe several different approaches, each of which has its own benefits and trade-offs.
 
+> [!NOTE]  
+> Versioning distributed systems and microservices isn't simple. It requires careful planning and using a proper versioning strategy.
+
```

Comment:
Again, please add more substance to this.  This isn't guidance yet, and a proper versioning strategy is the point of these sections you are adding this stuff into.  I'm going to close this PR. If you want to try again, feel free to do so by reopening or opening a new one.